Modules/_sha3/sha3module.c

```diff
@@ -122,6 +122,28 @@
 #define SHA3_squeeze Keccak_HashSqueeze
 #define SHA3_copystate(dest, src) memcpy(&(dest), &(src), sizeof(SHA3_state))
 
+typedef struct {
+    PyTypeObject *sha3_224_type;
+    PyTypeObject *sha3_256_type;
+    PyTypeObject *sha3_384_type;
+    PyTypeObject *sha3_512_type;
+    PyTypeObject *shake128_type;
```

Comment:
It's maybe to opportunity to introduce some consistency in variable name: func_bits_type. So rename shake128_type to shake_128_type and keccak224_type to keccak_224_type, for example?